bin/cgi_this

```diff
@@ -43,11 +46,15 @@ and runs every script in it as a CGI script.
 
 The following options are available:
 
-=over 4
+=over 5
 
 =item --port PORT
 
-Start the HTTP server ona specific C<PORT>. Default is 3000.
+Start the HTTP server on a specific C<PORT>. Default is 3000.
+
+=item --host IP_ADR
```

Comment:
Please can this be consistent with the way it's in the body of the text? Should be `IP_ADDRESS` too.

Reply:
Forgot :)